packages/web/docs/src/pages/features/sso-oidc-provider.mdx

```diff
@@ -55,3 +55,180 @@ Provider. Share this link with your users or add it to your SSO provider dashboa
 people will be redirected to it.
 
 E.g. `https://app.graphql-hive.com/auth/oidc?id=xxxxxxxx-1234-1234-1234-xxxxxxxxxxxx`
+
+## Configuring with Okta
+
+First we need to have an Okta account, so you can register for an account from
+https://login.okta.com/signin/register or sign up for a 30 days trial from
+https://www.okta.com/free-trial to ease the process of going through this guide.
+
+> Note: that unfortunately for sigining up for an Okta account, you're required to have a busieness
+> email.
```

Comment:
It's the requirement of Okta, we should skip it in our docs.

Reply:
ok